chromium relies on nightly crap and 1.85 has not replaced
this yet, so patch for now

--- a/build/rust/std/BUILD.gn
+++ b/build/rust/std/BUILD.gn
@@ -89,7 +89,7 @@ if (toolchain_has_rust) {
     # These are no longer present in the Windows toolchain.
     stdlib_files += [
       "addr2line",
-      "adler2",
+      "adler",
       "gimli",
       "libc",
       "memchr",
